flow-report/src/icons.tsx

```diff
@@ -94,3 +94,47 @@ export const CpuIcon: FunctionComponent = () => {
     </svg>
   );
 };
+
+export const HamburgerIcon: FunctionComponent = () => {
+  return (
+    <svg viewBox="0 0 18 12" width="18" height="12" role="img">
+      <rect width="18" height="2"></rect>
+      <rect y="5" width="18" height="2"></rect>
+      <rect y="10" width="18" height="2"></rect>
+    </svg>
+  );
+};
+
+export const Logo: FunctionComponent = () => {
+  return (
+    <svg class="lh-topbar__logo" height="24" width="24" viewBox="0 0 24 24">
+      <defs>
+        <linearGradient x1="57.456%" y1="13.086%" x2="18.259%" y2="72.322%" id="lh-topbar__logo--a">
```

Comment:
oh, these IDs would need unique prefixes too, hm maybe just move out of a generic icons to just topbar to avoid these issues?